daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/TypingSpec.scala

```diff
@@ -578,6 +578,19 @@ class TypingSpec extends WordSpec with TableDrivenPropertyChecks with Matchers {
           (\ (key: PositiveTestCase9:TBis) -> Cons @Party [(PositiveTestCase9:T {person} this), 'Alice'] (Nil @Party)  )
         } ;
       }
+
+      module PositiveTestCase10 {
+        record @serializable T (a: *) = {x: a};
+
+        // in the next line, expect T is first order.
```

Comment:
Is "first order" the right phrase here? I would have said "T must have kind *".

Reply:
fixed